src/condor_utils/dagman_utils.h

```diff
@@ -73,21 +73,96 @@ class dag_tokener {
 
 
 #include "enum.h"
+typedef std::list<std::string> str_list;
 
-namespace DagmanShallowOptions {
-
-  BETTER_ENUM(str, long,
-    ScheddDaemonAdFile = 0, ScheddAddressFile, ConfigFile, SaveFile
-  );
-
-  BETTER_ENUM(b, long,
-    PostRun = 0, DumpRescueDag, RunValgrind
-  );
+// Enum to represent booleans passed by command line
+enum class CLI_BOOL {
+	UNSET = 0,   // Boolean not set
+	TRUE,        // Boolean was set to True
+	FALSE        // Boolean was set to False
+};
```

Comment:
Consider making this `{ FALSE = 0, TRUE, UNSET }` (or maybe UNSET=1?) so that values correspond to what they usually are?